UnityProject/Assets/Scripts/Core/Input System/InteractionV2/Validations.cs

```diff
@@ -221,7 +221,7 @@ private static bool CanInteractByConsciousState(PlayerHealth playerHealth, bool
 
 		var playerObjBehavior = playerScript.pushPull;
 
-		if (!CanInteract(playerScript, side, allowSoftCrit, isPlayerClick: isPlayerClick))
+		if (false == CanInteract(playerScript, side, allowSoftCrit, isPlayerClick: isPlayerClick))
```

Comment:
```suggestion
		if ( CanInteract(playerScript, side, allowSoftCrit, isPlayerClick: isPlayerClick == false ))
```
it may work either way, but I think this is less of a sin (although still sinful, I really hate this part of the guidelines, lol)